external/src/MathUtils.cc

```diff
@@ -0,0 +1 @@
+#include "delphys/external/interface/MathUtils.h"
```

Comment:
if all defined in header, is this really needed?

Reply:
I didn't commit properly..